queries/rbs/highlights.scm

```diff
@@ -0,0 +1,42 @@
+["(" ")" "{" "}" "[" "]"] @punctuation.brackets
+["," "."]                 @punctuation.delimiter
+(method_signatures "|"    @punctuation.special)
+
+["class" "interface" "module" "end" "def" "alias"] @keyword.scope
+
+["=" "->" "<" "::"]           @operator
+(supertype)                   @operator
+(union_type "|")              @operator
+(intersection_type "&")       @operator
+[":" "?"]                     @operator.special
+(splat_parameter "*")         @operator.special
+(double_splat_parameter "**") @operator.special
+
+["type" (visibility) (unchecked) (variance)] @keyword
```

Comment:
Move "type" with the other keywords. The rest should probably be `@type.qualifier`.